MAUI/Calendar/selections.md

```diff
@@ -0,0 +1,249 @@
+Eg: 
+* In the year view, when the range is selected as Sep - Dec, then the range value will be 01-09-2022 to 31-12-2022.
+* In the decade view, when the range is selected as 2022 - 2025, then the range value will be 01-01-2022 to 31-12-2025.
+* In the century view, when the range is selected as 2020-2029 to 2030-2039, then the range value will be 01-01-2020 to 31-12-2039.
+
+### Range Selection Direction
+The Range Selection Direction is specified in the calendar property enumeration which is uesd to select the range based on the direction. The `RangeSelectionDirection` provides five types of direction such as `Default`, `Forward`, `Backward`, `Both` and `None`. The default `RangeSelectionDirection` is `Default` in the SfCalendar.
+
+#### Range Selection Direction - Forward
+If the `RangeSelectionDirection` property to `Forward` allows you to select only the dates after the selected range start date and the dates before the range start date is considered as disabled date. The start date will not be changed.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+<calendar:SfCalendar  x:Name="Calendar" 
+                        View="Month"
+                        SelectionMode="Range"
+                        RangeSelectionDirection="Forward">
+</calendar:SfCalendar>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+this.Calendar.SelectionMode = CalendarSelectionMode.Range;
+this.Calendar.RangeSelectionDirection = CalendarRangeSelectionDirection.Forward;
+
+{% endhighlight %}
+{% endtabs %}
+
+![Forward range selection in .NET MAUI Calendar.](images/selection/maui-forward-range-selection.png)
+
+#### Range Selection Direction - Backward
+If the `RangeSelectionDirection` property to `Backward` allows you to select only the dates before the selected range end date and the dates after the range end date is considered as disabled date. The end date will not be changed.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+<calendar:SfCalendar  x:Name="Calendar" 
+                        View="Month"
+                        SelectionMode="Range"
+                        RangeSelectionDirection="Backward">
+</calendar:SfCalendar>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+this.Calendar.SelectionMode = CalendarSelectionMode.Range;
+this.Calendar.RangeSelectionDirection = CalendarRangeSelectionDirection.Backward;
+
+{% endhighlight %}
+{% endtabs %}
+
+![Backward range selection in .NET MAUI Calendar.](images/selection/maui-backward-range-selection.png)
+
+#### Range Selection Direction - Both
+If the `RangeSelectionDirection` property to `Both` allows you to extend the selection from the selected range. Then the tapped date is considered based on (if the date is nearby start date then start date is updated, else if the date is nearby end date then end date is updated, else if in-between the start and end date and both have same distance then the start date is updated.
```

Comment:
If you set the `RangeSelectionDirection` property to `Both,` you can extend the selection from the selected range. Then, the tapped date is considered based on the (if the date is nearby the start date, then the start date is updated, else if the date is nearby the end date, then the end date is updated, else if in-between the start and end date and both have the same distance, then the start date is updated).

Reply:
Content modified